MWContentDisplayPlugin/MWContentDisplayPlugin/ContentStackStep/Remote/MWNetworkContentDisplayStackViewController.swift

```diff
@@ -30,16 +30,6 @@ class MWNetworkContentDisplayStackViewController: MWContentDisplayStackViewContr
     }
     
     // MARK: RemoteContentStepViewController
-    func loadContent() {
-        self.showLoading()
-        self.remoteContentStep.loadContent { [weak self] result in
-            self?.hideLoading()
-            switch result {
-            case .success(let items): self?.update(content: items)
-            case .failure(let error): self?.show(error) { [weak self] in self?.goBackward() }
```

Comment:
@FWJulien Do we need to preserve the `goBackward()` alert dismiss action? I don't believe it's included in the default `loadContent()` implementation/flow.

It's a bit awkward, but we could add a `handleErrorDismiss()` to `RemoteContentStepViewController` protocol, and have a default implantation that does nothing?

Reply:
@FWJonathan I think you're right, we probably want the goBackward because else user will be stuck on an empty screen if we failed to load the content? So we should make `func handleError(_ error: Error)` public and add the capability to pass a block to be executed on alert dismiss, agreed?

Reply:
I'm not sure quite what you have in mind with the alert dismiss block, but go for it if you think it will work and I will review it.

Reply:
@FWJonathan I have updated the implementation so that the controllers uses the handleError function so that 401 are intercepted, and changed it to load its content on view will appear using the ContentClearable pattern.